kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeSidePublicTab.js

```diff
@@ -412,7 +405,11 @@ define([
 
             this.resultPanel = $('<div role="table" data-test-id="result">');
 
-            this.resultsFooterMessage = $('<div>');
+            this.resultsFooterMessage = $('<div>')
+                .css('display', 'flex')
+                .css('flex-direction', 'row')
+                .css('align-items', 'center')
+                .css('justify-content', 'center');
```

Comment:
how did this inline css sneak in? 😞

Reply:
That was just reformatted from how it was before. I think it'll have to be part of a greater effort...